src/cascadia/TerminalSettingsModel/ActionMap.cpp

```diff
@@ -336,43 +336,43 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         }
         else
         {
-            // We're adding an action that already exists in our layer.
+            // This is an action that we already have a mutable "masking" record for.

```

Comment:
but this isn't.. a masking record. is it? this is just one that *originated* in our layer. This comment was meant for line R374 ;P